modules/affile/wildcard-source.c

```diff
@@ -310,6 +312,21 @@ _init_filename_pattern(WildcardSourceDriver *self)
   return TRUE;
 }
 
+static gboolean
+_init_exclude_pattern(WildcardSourceDriver *self)
+{
+  if (self->exclude_pattern) {
```

Comment:
Style check fails on this, please correct

<img width="633" height="505" alt="Screenshot 2025-07-15 at 8 44 34" src="https://github.com/user-attachments/assets/c83ad717-838e-481d-a69f-5048be7d0165" />